ruby/ext/google/protobuf_c/repeated_field.c

```diff
@@ -466,6 +441,17 @@ VALUE RepeatedField_plus(VALUE _self, VALUE list) {
   return dupped;
 }
 
+VALUE RepeatedField_concat(VALUE _self, VALUE list) {
```

Comment:
using `RepeatedField_plus` for concat is not correct.  `Array#concat` appends to `self`, while  `RepeatedField_plus` created a new `RepeatedField` object